core/cloudflow-operator/src/main/scala/cloudflow/operator/action/AppActions.scala

```diff
@@ -69,23 +70,25 @@ object AppActions {
       )
     )
 
-  private def akkaRole(namespace: String, labels: CloudflowLabels): Role =
+  private def akkaRole(namespace: String, labels: CloudflowLabels, ownerReferences: CloudflowOwnerReferences): Role =
     Role(
       metadata = ObjectMeta(
         name = Name.ofAkkaRole(),
         namespace = namespace,
-        labels = labels(Name.ofAkkaRole)
+        labels = labels(Name.ofAkkaRole),
+        ownerReferences = ownerReferences.list
```

Comment:
yeah I think it will be easier just to pass along `List[OwnerReference]` I don't think we really need to `CloudflowOwnerReferences` type.

Reply:
Removed it.